### ISSUE TITLE
How should I use my dataset

### ISSUE THREAD

**Issue by dcdethan** [zjuela/LapSRN-tensorflow#5]

**Comment by zjuela** [comment]
For your own data, you can 
1) generate HR-LR pairs and put them separately in two folders, LR_folder and HR_folder, with the same name. 
2) modify config.py file with your LR/HR folder_path.

**Comment by dcdethan** [comment]
Thank you for your reply,I try it now.

**Comment by dcdethan** [comment]
Hi,you use the validation set, do you need test sets?

**Comment by zjuela** [comment]
I did not implement evaluation code, but simply test code for single image. It should be easy to implement in a similar structure.

**Comment by dcdethan** [comment]
ok,thanks a lot.

**Comment by zymize** [comment]
do you use test sets in this code? @aipeteryao
